test/replace-state-patch.html

```diff
@@ -0,0 +1,81 @@
+<script>
```

Comment:
This script seems like a good candidate for wrapping in an IIFE; no implicit globals and you could remove some of the `__` prefixing.

Reply:
Is the only thing you intend to 'export' `cooldownFunction`?

Reply:
implemented